app/src/main/java/agency/five/codebase/android/movieapp/ui/component/UserScoreProgressBar.kt

```diff
@@ -0,0 +1,83 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.ui.theme.MovieAppTheme
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.DrawStyle
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+
+
+@Composable
+fun UserScoreProgressBar(
+    percentage: Float,
+    radius: Dp = 80.dp,
+    animationDuration: Int = 1000,
+) {
+    var animFinished by remember {
+        mutableStateOf(false)
+    }
+
+    val currentPercent = animateFloatAsState(
+        targetValue = if (animFinished) percentage else 0f,
+        animationSpec = tween( //ugradena animacija u kojoj se odreduje duljina, cekanje...
```

Comment:
use only english language in code - other developers maybe would not understand your comments if you're using any other language

Reply:
Done